bin/eslint.js

```diff
@@ -16,6 +16,8 @@ var cli = new CLIEngine(options);
 var debug = false;
 var checks = require("../lib/checks");
 
+console.error("ESLint is running with the " + cli.getConfigForFile(null).parser + " parser.");
```

Comment:
Got it. Makes sense given the current state of the code not visible in the diff.